rentals.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_reader :book, :person
+  attr_accessor :date
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+    book.rentals << self
```

Comment:
- [ ] This line is failing and it's crashing the project when I try to create a rental for a book, would check into it?
![Screenshot from 2022-08-25 12-07-08](https://user-images.githubusercontent.com/56886064/186637691-726a1263-cc4e-4d89-8f9d-7e837d3338d2.png)

Reply:
This issue is well resolved! ✔️